tf_encrypted/convert/convert.py

```diff
@@ -279,21 +286,30 @@ def find_leaves(scope, subscope_map):
   return input_leaves, output_leaves
 
 
-def match_numbered_scope(specop, search_string, return_group=True):
+def match_numbered_scope(specop, search_string,
+                         return_group=True,
+                         not_numbered=False):
```

Comment:
To make this slightly more readable, I'd suggest renaming this to `numbered` and flipping the truth value to avoid the double negative here

Reply:
@jvmancuso Done.